### PR TITLE
Fix 4684 update geostory guide

### LINKED ISSUE #4684: Geostory user guide updating
## Description

Some improvement and implementation for the Exploring Stories section in uesrguide:

- [x] Owerview updating
- [x] Share implementation
- [x] Web Page Section implementation 
- [x] Story setting implementation
- [x] Configure map implementation
- [x] Title section, Paragraph Section, Media Section and Immesrive Section modification with Media content and Media background

Documentation section involved*
- [x] User Guide
- [ ] Developer Guide

### PR DESCRIPTION
**Pull request by ElenaGallo**
## Description

with this PR:
 - Owerview updating with Web Page Section, Top Bar and  Share
 - Web Page Section implementation
 - Story setting implementation
 - Configure map implementation
 - Title section, Paragraph Section, Media Section and Immesrive Section modification with Media content and Media background

Fix #4684

### CODE REVIEW

File: docs/user-guide/title-section.md [hunk 1]
```diff
@@ -71,4 +71,16 @@ In this case the background editor toolbar allow the user to customize the image
...
\ No newline at end of file
+    The *Align content* and the *Change field theme* buttons are disabled if the image size is full screen
+
+### Maps
+
+In this case, adding a map as background, the result will be like this:
+
+<img src="../img/title-section/map-background.jpg" class="ms-docimage"/>
+
+The background toolbar, in case of the map, has a new button:
```
Comment (francoperversi) [inline docs/user-guide/title-section.md]: Specific that is new compared to the images one. Compared to the images one one button is also missing

File: docs/user-guide/web-section.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+<img src="../img/web-section/web-window.jpg" class="ms-docimage"/>
+
+An example of a new Web Page Section with a web page added can be like the following:
+
+<img src="../img/web-section/web-page.jpg" class="ms-docimage"/>
+
+Once the first web page is added it is possible to add new web page, text and media content or remove the existing ones, just as it is explained in the [Paragraph Sections](paragraph-section.md).
+
+<img src="../img/web-section/add-section.jpg" class="ms-docimage"/>
+
```
Comment (francoperversi) [inline docs/user-guide/web-section.md]: toolbar description missing

File: docs/user-guide/media-section.md [hunk 3]
```diff
@@ -5,20 +5,6 @@ Media Section are similar to [Paragraph Sections](paragraph-section.md) but the
 
 <img src="../img/media-section/media-s.jpg" class="ms-docimage"/>
 
-Once the first media is added to the Media Section it is possible to add new media or text content or remove the existing ones, just like in the [Paragraph Sections](paragraph-section.md). 
+Once the first media is added to the Media Section it is possible to add new media, text content or web page content or remove the existing ones, just like in the [Paragraph Sections](paragraph-section.md). 
 
-<img src="../img/media-section/media-add-c.jpg" class="ms-docimage" style="width:400px"/>
```
[the remark this answers is not in the document]
Comment (ElenaGallo) [reply docs/user-guide/media-section.md]: Talking to @tdipisa we decided not to report, in this section, those functions that are the same as in the paragraph section but to report only the link of the paragraph section.

File: docs/user-guide/story-setting.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+
```
[the remark this answers is not in the document]
Comment (ElenaGallo) [reply docs/user-guide/story-setting.md]: I put the Story Setting in a separare section because I don't think it should be in the overview section

**Pull request closed without merging**